Client/metrics.yaml

```diff
@@ -600,7 +600,7 @@ legacy.ids:
 
 deletion:
   fxa_device_id:
-    type: uuid
+    type: string
```

Comment:
Naming nit: calling this "fxa device id" risks confusing our future selves into thinking that it should be the server-provided FxA device id. Would it be possible to call this field `sync_device_id` or `fxa_hashed_device_id` or similar? This would avoid overloading a term that means something different in an adjacent mental context - if you say "fxa device id" anywhere else but in reference to telemetry pings, it means the server-provided id.

(The existing code is not great in this respect, as we all just found out...but it would be good to do better here if possible).

Reply:
On desktop we call this field [`sync_device_id`](https://searchfox.org/mozilla-central/rev/ff82c973f8ccb0475ec32439e9ec07014b3a681f/services/sync/modules/telemetry.js#803) and it may be worth using that for consistency here.

Reply:
changed name to `sync_device_id:`